cargo-fix/tests/all/subtargets.rs

```diff
@@ -28,7 +28,7 @@ fn fixes_missing_ampersand() {
 [COMPILING] foo v0.1.0 (CWD)
 [FINISHED] dev [unoptimized + debuginfo]
 ";
-    p.expect_cmd("cargo fix --all-targets").stdout("").stderr(stderr).run();
+    p.expect_cmd("cargo fix -- --all-targets").stdout("").stderr(stderr).run();
```

Comment:
Hm, do you want to strictly separate which args are "cargo args" like `--all-targets` or `--features`, or do you eventually want to add args to cargo-fix that map to cargo args?